inc/class-job.php

```diff
@@ -23,17 +23,19 @@ class Job {
 	protected $db;
 	protected $table_prefix;
 
+	/**
+	 * @var null|bool
+	 */
+	static protected $blogs_table_exists;
```

Comment:
Visibility's always first.
```suggestion
	protected static $blogs_table_exists;
```